donation/models.py

```diff
@@ -22,7 +22,7 @@ class Donation(models.Model):
     donor_address = models.CharField(max_length=200, verbose_name="Dirección")
     donor_email = models.EmailField(verbose_name="Correo Electrónico")
     ong = models.ForeignKey(
-        Ong, on_delete=models.CASCADE, related_name='donacion', verbose_name="ONG")
+        Ong, on_delete=models.CASCADE, related_name='donacion', verbose_name="ONG",default=1, blank=True)
```

Comment:
You really shouldn't modify the model like this. You need the NGO to be created so... why don't you just create a superuser first?
This way, you can specify with decorators that login is required, and maybe make another decorator like I did on #347 and #348 to specify which NGO is required. Anyway, you just need the login required.